extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -57,6 +57,12 @@ amp-ad[type="adsense"] > iframe {
   transform: translate(-50%, -50%);
 }
 
+[dir=rtl] amp-ad[data-a4a-upgrade-type="amp-ad-network-doubleclick-impl"] > iframe,
+[dir=rtl] amp-ad[type="adsense"] > iframe {
+  left: 0 !important;
```

Comment:
I think we need `left: 0% !important` here. I tested with `left : 0` and `left: 0%` and the output is different. Do you know why?

Reply:
ignore me.....

Reply:
discussed offline